app/graph/mutations/annotation_mutations.rb

```diff
@@ -1,20 +1,23 @@
 module AnnotationMutations
-  create_fields = {
-    content: '!str',
-    annotation_type: '!str',
-    annotated_id: 'str',
-    annotated_type: 'str',
-    locked: 'bool'
-  }
+  MUTATION_TARGET = 'annotation'.freeze
+  PARENTS = ['source', 'project_media', 'project', 'task'].freeze
 
-  update_fields = {
-    content: 'str',
-    annotation_type: 'str',
-    annotated_id: 'str',
-    annotated_type: 'str',
-    assigned_to_ids: 'str',
-    locked: 'bool'
-  }
+  module SharedCreateAndUpdateFields
+    extend ActiveSupport::Concern
 
-  Create, Update, Destroy = GraphqlCrudOperations.define_crud_operations('annotation', create_fields, update_fields, ['source', 'project_media', 'project', 'task'])
+    included do
```

Comment:
TODO: I think this can all be moved into the Create function, for clarity sake, since we don't have/use an Update

Reply:
TODO: Check if this should be included in future annotation module